code/modules/mob/living/carbon/human/human.dm

```diff
@@ -395,7 +395,7 @@
 	. = ..()
 	if(!. && (injection_flags & INJECT_TRY_SHOW_ERROR_MESSAGE) && user)
 		var/obj/item/bodypart/the_part = get_bodypart(target_zone || user.zone_selected)
-		to_chat(user, span_alert("There is no exposed flesh or thin material on [p_their()] [the_part.name]."))
+		to_chat(user, span_alert("There is no exposed flesh or thin material on [isnull(the_part) ? "them" : "[p_their()] [the_part.name]"]."))
```

Comment:
i'm not a fan of this since "them" implies the body as a whole, where as this is targeting a specific zone of their body.

Reply:
would definitely prefer something more along the lines of "they don't have an arm to inject!" or something more explicit about why it failed instead.

Reply:
the issue here was precise zones not missing limbs, so i just made it generalize the zone instead.

Reply:
nvm